bumble/pandora/config.py

```diff
@@ -35,6 +36,12 @@ def load_from_dict(self, config: Dict[str, Any]) -> None:
             'io_capability', 'no_output_no_input'
         ).upper()
         self.io_capability = getattr(PairingDelegate, io_capability_name)
+        identity_address_type_name: str = config.get(
```

Comment:
Ah, my bad. If you think `None` is a better default this need to be worked out a bit

Reply:
Actually I think that's a good idea to force setting a type when using Pandora, otherwise the default behavior is a little bit ambiguous which implies the device has 2 identity address. The thing left is I should remove `Optional`.